packages/components/psammead-story-promo/src/IndexAlsos/__snapshots__/index.test.jsx.snap

```diff
@@ -180,6 +182,7 @@ exports[`Index Alsos should render multiple correctly 1`] = `
         role="listitem"
       >
         <a
+          aria-labelledby="IndexAlsosLink-/hausa/labarai-42837051"
```

Comment:
I am not sure, need to investigate. But I think it will require us to predict what the children of the Anchor elements are. This will add another layer of complexity.

Reply:
I think we have similar cases with other PRs where this happened and we decided that despite not needing it if it does not cause any issue we will keep it (Not sure about this, will need to double check with @nicola-d-bbc  )

Reply:
How much does it add to the page weight? This would be on every promo on the home page for example?

Reply:
I tried to find a way to mesure this, asked further help to Andrew to be extra sure. We have no way to mesure this, but I assume it won't have a major impact as this is only on every Alsos Link. We don't use many of these link in our story promos. I will check after we merged this Psammead change just to be extra safe.